CHANGES_UNRELEASED.md

```diff
@@ -37,7 +37,9 @@
 - iOS: Added `FxAccountManager.getSessionToken`. Note that you should request the `.session` scope in the constructor for this to work properly. ([#2638](https://github.com/mozilla/application-services/pull/2638))
 - iOS: Added `FxAccountManager.getManageAccountURL`. ([#2658](https://github.com/mozilla/application-services/pull/2658))
 - iOS: Added `FxAccountManager.getTokenServerEndpointURL`. ([#2658](https://github.com/mozilla/application-services/pull/2658))
-
+- iOS: Added migration methods to `FxaAccountManager` ([#2637](https://github.com/mozilla/application-services/pull/2637)):
```

Comment:
This says `FxaAccountManager`, but the changelog entries above it say `FxAccountManager`. The class seems to be named `FxaAccountManager` but it lives in a file named `FxAccountManager.swift`, and grep tells me that some of the comments in `FxAccountDeviceConstellation.swift` refer to it as `FxAccountManager`.

I think we should call it `FxAccountManager` consistently throughout, by analogy to the existing `FxAccount` class. I'd also be fine with consistent use of `FxaAccountManager`. But please consider doing a bit of cleanup for consistency either way, while you're in here.